src/test/unit_tests/test_bucketspace.js

```diff
@@ -167,6 +167,64 @@ mocha.describe('bucket operations - namespace_fs', function() {
         }
         assert.ok(fs_utils.file_must_not_exist(tmp_fs_root));
     });
+
+    mocha.it('delete account by uid, gid', async function() {
+        let read_account_resp1 = await rpc_client.account.read_account({ email: 'account_wrong_uid@noobaa.com' });
+        assert.ok(read_account_resp1);
+
+        // create another account with the same uid gid
+        let account_wrong_uid1 = await rpc_client.account.create_account({...new_account_params,
+                email: 'account_wrong_uid1@noobaa.com',
+                name: 'account_wrong_uid1',
+                nsfs_account_config: {
+                    uid: 26041992,
+                    gid: 26041992,
+                }
+            }
+        );
+        console.log(inspect(account_wrong_uid1));
+        assert.ok(account_wrong_uid1);
+        await rpc_client.account.delete_account_by_property({
+                nsfs_account_config: {
+                    uid: 26041992,
+                    gid: 26041992,
+                }
+            }
+        );
+        // check that both accounts deleted
+        try {
+            let deleted_account_exist = await rpc_client.account.read_account({ email: 'account_wrong_uid@noobaa.com' });
+            assert.fail(`found account: ${deleted_account_exist} - account should be deleted`);
+        } catch (err) {
+            assert.ok(err.rpc_code === 'NO_SUCH_ACCOUNT');
+        }
+
+        try {
+            let deleted_account_exist = await rpc_client.account.read_account({ email: 'account_wrong_uid1@noobaa.com' });
+            assert.fail(`found account: ${deleted_account_exist} - account should be deleted`);
+        } catch (err) {
+            assert.ok(err.rpc_code === 'NO_SUCH_ACCOUNT');
+        }
+        let list_account_resp2 = await rpc_client.account.list_accounts();
+        console.log(inspect(list_account_resp2));
```

Comment:
Do we want to do something with this list? 
If not, then why are we listing?

Reply:
I think the idea was to verify we don't break anything after deletion, but Romy can correct me if I'm wrong

Reply:
In that case, We probably need an assert to verify it.

Reply:
so it was for my verification since the first try catch assertion is probably enough for this test but I added another assert for not deleting all accounts for example.